### PR TITLE
Add `No commit yet` empty state for "Last commit"

### PR DESCRIPTION
**Pull request by tupe12334**
closes: #3597 

## PR Details

This PR adds not commit yet message
Screen Shot 2022-08-29 at 11 42 29

## PR Checklist
- [x] Tests for the changes have been added
- [x] `npm test` doesn't throw any error

IMPORTANT: Please review the [CONTRIBUTING.md](https://github.com/amplication/amplication/blob/master/CODE_OF_CONDUCT.md) file for detailed contributing guidelines.

### CODE REVIEW

File: packages/amplication-client/src/Workspaces/ResourceListItem.scss [hunk 1]
```diff
@@ -26,16 +26,20 @@
...
     }
   }
 
   &__last-build {
     font-weight: normal;
+    &__title {
+      color: var(--black100);
+    }
+    &__not-yet {
+      color: --var(--black80);
```
Comment (abrl91) [inline packages/amplication-client/src/Workspaces/ResourceListItem.scss:42]:
```suggestion
      color: var(--black80);
```

File: packages/amplication-client/src/Workspaces/ResourceListItem.scss [hunk 2]
```diff
@@ -26,16 +26,20 @@
...
-
     .user-and-time {
       margin-left: var(--default-spacing-small);
     }
   }
 
   &__last-build {
     font-weight: normal;
+    &__title {
+      color: var(--black100);
```
Comment (abrl91) [inline packages/amplication-client/src/Workspaces/ResourceListItem.scss:39]: Why did you add it? you already have it (line 15)
image

Comment (tupe12334) [reply packages/amplication-client/src/Workspaces/ResourceListItem.scss:39]: It's a different type of title, it's the panel title.
